### DISCUSSION TITLE
JS Validation on extended WP object forms

### DISCUSSION THREAD

**Discussion by ghost** [pods-framework/pods#6611] [Ideas]
## Tasks
- [ ] Look into pulling our existing JS form validation code that we use on ACT forms
- [ ] On WP forms, call JS function/method once page loads that hooks into validation

---

Original message:

When an input is not valid, like blank input for a required field, letters in a number only field, can we have the validation message on the same page rather than going to another page to see the error message and then come back. Because if I click the back button to do the amends, I'm actually creating another page rather than editing the page I've worked on.

Thanks.

**Comment by jamesgol** [comment]
Some of the code is supposed to do this already but it doesn't work.  I fixed it here #2357 but it's on hold right now.  You can try out that code and comment there if it works for you

**Comment by jamesgol** [comment]
The other pull request was commited, so this should be resolved.

**Comment by sc0ttkclark** [comment]
Updated issue title and added tasklist to complete

**Comment by sc0ttkclark** [comment]
In Pods 2.7 we are now showing back links when using wp_die(). This isn't a fix for this issue however it does at least improve things in a minor way.

**Comment by sc0ttkclark** [comment]
Moving forward in https://github.com/pods-framework/pods/issues/6958
